### PR TITLE
Add test for canceling a running compose

### PR DESCRIPTION
**Pull request by jrusz**
This is a modification of an existing test for compose sanity.
Cherry-picked from master.

### CODE REVIEW

**Comment by jrusz** [comment]
Oh, didn't know that, should be ok now.

**Comment by atodorov** [comment]
1656691 is CLOSED, you need another one which is approved for the current release before we can merge.

**Comment by jrusz** [comment]
I got another related rhbz#1788461. This one should be ok.

File: tests/cli/test_compose_sanity.sh [hunk 1]
```diff
@@ -22,6 +22,26 @@ rlJournalStart
     rlPhaseEnd
 
     rlPhaseStartTest "compose start"
+        UUID=`$CLI compose start example-http-server ami`
+        rlAssertEquals "exit code should be zero" $? 0
+        UUID=`echo $UUID | cut -f 2 -d' '`
+
+        if [ -n "$UUID" ]; then
+            until $CLI compose info $UUID | grep 'RUNNING'; do
```
Comment (bcl) [inline tests/cli/test_compose_sanity.sh]: There is some possibility that this could return FAILED and then it would be stuck in the loop.

Comment (jrusz) [reply tests/cli/test_compose_sanity.sh]: Thanks for noticing this. I added a check for FAILED compose. I guess this should go to master branch too.

**Pull request merged**